cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -86,12 +91,54 @@ func (m *Manager) Deploy(ctx context.Context) (*DeployResult, error) {
 		return nil, fmt.Errorf("updating environment with deployment outputs: %w", err)
 	}
 
+	stAccount, hasBindMountOperation := m.env.LookupEnv(operationsBindMount)
+	if hasBindMountOperation {
+		for key, value := range m.env.Dotenv() {
+			if key != operationsBindMount && strings.HasPrefix(key, operationsBindMountPrefix) {
+				valueParts := strings.Split(value, ",")
+				if len(valueParts) != 2 {
+					return nil, fmt.Errorf("invalid bind mount operation value: %s", value)
+				}
+				fileShareName, source := valueParts[0], valueParts[1]
+				if err := bindMountOperation(
+					ctx, m.fileShareService, m.env.GetSubscriptionId(), stAccount, fileShareName, source); err != nil {
+					return nil, fmt.Errorf("error binding mount: %w", err)
+				}
+			}
+		}
+	}
+
 	// make sure any spinner is stopped
 	m.console.StopSpinner(ctx, "", input.StepDone)
 
 	return deployResult, nil
 }
 
+func bindMountOperation(
+	ctx context.Context,
+	fileShareService storage.FileShareService,
+	subId, storageAccount, fileShareName, source string) error {
+
+	shareUrl := fmt.Sprintf("https://%s.file.core.windows.net/%s", storageAccount, fileShareName)
```

Comment:
May need to take into account `cloud.Cloud`

Reply:
done